reflex/components/radix/primitives/accordion.py

```diff
@@ -493,6 +515,11 @@ def create(
         # The item requires a value to toggle (use a random unique name if not provided).
         value = props.pop("value", get_unique_variable_name())
 
+        if "AccordionItem" not in (
+            cls_name := props.pop("class_name", "AccordionItem")
+        ):
+            cls_name = f"{cls_name} AccordionItem"
+
         if (header is not None) and (content is not None):
             children = [
                 AccordionHeader.create(
```

Comment:
we can simplify the following code now: removing the `class_name` specification and replacing `Icon` with the new `AccordionIcon`.

people looking to use the low level API will probably copy this as an example

Reply:
sure let's do that